<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PGetNonMusrRunBlockDialog</class>
 <widget class="QDialog" name="PGetNonMusrRunBlockDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Get NonMusr Run Block Parameters</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../musredit.qrc">
    <normaloff>:/images/musrfit.xpm</normaloff>:/images/musrfit.xpm</iconset>
  </property>
  <widget class="QGroupBox" name="fRunHeaderInfo_groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>401</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Run Header Info</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>381</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="fRunFileName_label">
       <property name="text">
        <string>Run File Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fRunFileName_lineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>60</y>
      <width>381</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="fBeamline_label">
       <property name="text">
        <string>Beamline</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>23</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="fBeamline_lineEdit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>91</y>
      <width>381</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="fInstitute_label">
       <property name="text">
        <string>Institute</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>34</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="fInstitute_comboBox">
       <item>
        <property name="text">
         <string>PSI</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RAL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TRIUMF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JPARC</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="fFileFormat_label">
       <property name="text">
        <string>File Format</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fFileFormat_comboBox">
       <item>
        <property name="text">
         <string>ASCII</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DB</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="fReqiredEntries_groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>401</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Required Entries</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>381</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="fMap_label">
       <property name="text">
        <string>Map</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fMap_lineEdit">
       <property name="text">
        <string>0  0  0  0  0  0  0  0  0  0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>60</y>
      <width>281</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="fXYData_label">
       <property name="text">
        <string>xy-Data</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>37</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="fXData_lineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="fYData_lineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>92</y>
      <width>291</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="fFitRange_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Fit Range</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fFitRangeStart_lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fFitRangeEnd_lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>401</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="fHelp_pushButton">
      <property name="text">
       <string>&amp;Help</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="fOk_pushButton">
      <property name="text">
       <string>&amp;OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="fCancel_pushButton">
      <property name="text">
       <string>&amp;Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>fRunFileName_lineEdit</tabstop>
  <tabstop>fBeamline_lineEdit</tabstop>
  <tabstop>fInstitute_comboBox</tabstop>
  <tabstop>fFileFormat_comboBox</tabstop>
  <tabstop>fMap_lineEdit</tabstop>
  <tabstop>fXData_lineEdit</tabstop>
  <tabstop>fYData_lineEdit</tabstop>
  <tabstop>fFitRangeStart_lineEdit</tabstop>
  <tabstop>fFitRangeEnd_lineEdit</tabstop>
  <tabstop>fOk_pushButton</tabstop>
  <tabstop>fCancel_pushButton</tabstop>
  <tabstop>fHelp_pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../musredit.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fCancel_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PGetNonMusrRunBlockDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fOk_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PGetNonMusrRunBlockDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fHelp_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PGetNonMusrRunBlockDialog</receiver>
   <slot>helpContent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>helpContent()</slot>
 </slots>
</ui>
